### PR TITLE
Fixed and cleaned up create functions

### PR DESCRIPTION
**Pull request by loshprung**
- Removed redundant `connect.php`
- Fixed broken create functions
- Consolidated new_account and new_task into a single file `create.php`

This commit is needed to fix bugs that should not have gotten past testing, so this is a reminder to ***please test your new changes before making a pull request***

### CODE REVIEW

File: backend/create.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+function create_task($json_hash){
+	GLOBAL $db;
+	$valid = true;
+	$account_id = "";
+	$tasks_name = "";
+	$category = "";
+    $deadline = "";
+    $priority = "";
```
Comment (maddiewaldie) [inline backend/create.php]: Should these also be indented?
```	suggestion
    	$deadline = "";
    	$priority = "";
```

File: backend/create.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+	// list of fields that must not be null
+	$fields = [
+		"account_id",
+		"tasks_name",
+		"category",
+    	"deadline",
+    	"priority"
```
Comment (maddiewaldie) [inline backend/create.php]: Should these also be indented?
```suggestion
    		"deadline",
    		"priority"
```

**Review by maddiewaldie — APPROVED** [review]
Looks good to me! Just a few minor indentation comments.

**Comment by loshprung** [comment]
Yeah, I think this is inconsistency between using tabs and spaces for indentation. Will amend

**Review by markc331 — APPROVED** [review]

**Pull request merged**